repository/server/RequestWorker.cpp

```diff
@@ -279,10 +281,11 @@ RequestWorker::procDataGetSizeRequest()
         }
         else
         {
-            data_sz->set_size( 0 );
```

Comment:
Why did you remove this line?

Reply:
Odd, this is not showing the latest code changes, but it was replaced with an exception due to not finding the specified file path. The prior code would allow a transfer to succeed if there was a repo issue.

Reply:
Tyler and I noticed the other day that GitHub diffs were not showing the most recent changes perhaps this is the same problem.